tests/src/Interop/PInvoke/SizeParamIndex/PInvoke/Invalid/InvalidParamIndex.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <DefineConstants>$(DefineConstants);STATIC</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="InvalidParamIndex.cs" />
+  </ItemGroup>
+  <Import Project="../../../../Interop.settings.targets" />
```

Comment:
Please change this to `<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Interop.settings.targets))\Interop.settings.targets" />`